packages/@aws-cdk/cloud-assembly-schema/schema/cloud-assembly.metadata.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think this file should be called `cloud-assembly.version.json` for discoverability and in order to avoid abuse.

Reply:
👍